### DISCUSSION TITLE
Pretix Update check showing version 3.18 as latest version

### DISCUSSION THREAD

**Discussion by Daniel-Koulouris** [pretix/pretix#2156] [Q&A] [answered]
Not sure if this is local to my setup or possibly a bug but the update check will never notify me as the latest version is stuck on 3.18 always
![image](https://user-images.githubusercontent.com/20973998/126442141-776a2952-40fc-47a3-99f2-f7f3ce9bdbb0.png)

**Comment by raphaelm** [comment, marked as answer]
Ouch, I think that's us. WIll fix today.
